src/components/Insight/Comments.js

```diff
@@ -58,24 +63,44 @@ function editComment (id, content) {
   })
 }
 
-export default ({ id, authorId, count }) => {
+export default ({
+  id,
+  authorId,
+  count,
+  onEdit = editComment,
+  onDelete = deleteComment,
+  onCreate = createComment,
+  onGet = getComments
+}) => {
   const [comments, setComments] = useState([])
   const [loading, setLoading] = useState(true)
 
-  useEffect(() => {
-    getComments(id).then(({ data }) => {
+  const loadCommentsCount = () => {
```

Comment:
`loadCommentsCount` is a misleading name, it implies the process of getting a single value – **comments count**. Meanwhile, array of comments is being fetched.